### PR TITLE
Es/bubble image

### PR DESCRIPTION
**Pull request by ErikSin**
# Bubble Sync as static image

Shows different images based on whether sync has started or not.

image

image

### CODE REVIEW

File: src/screens/sync/DevicesList.tsx [hunk 1]
```diff
@@ -34,16 +34,18 @@ export const DevicesList = ({
...
   onDevicePress: (isNotIdle: boolean, peer: Peer) => void;
   onInfoPress: () => void;
+  show: boolean;
 }) => {
   const { formatMessage: t } = useIntl();
   const [shouldStart, setShouldStart] = React.useState(false);
 
   return (
-    <View>
+    <View style={show ? undefined : { display: "none" }}>
```
Comment (ErikSin) [inline src/screens/sync/DevicesList.tsx:48]: The progress state is held locally. So everytime we unmounted this (everytime we opened the bubbles page), it caused the  sync to start again. This is hacky, but we can't lift the state of the progress without having performance issues.

File: src/screens/sync/Devices.tsx [hunk 2]
```diff
@@ -69,16 +70,20 @@ export const Devices = ({ mode, role }: { mode: ViewMode; role: Role }) => {
...
-          <DevicesList
-            peers={peers}
-            onInfoPress={() => {
-              setModalMode({ type: "info" });
-              openSheet();
-            }}
-            onDevicePress={handleDevicePress}
-          />
-        ) : null}
+        ) : (
```
Comment (ErikSin) [inline src/screens/sync/Devices.tsx:73]: see: https://github.com/digidem/mapeo-mobile-shell/pull/36/files#r1173088716

File: src/screens/sync/DevicesList.tsx (hunk 1 above)
Comment (achou11) [reply src/screens/sync/DevicesList.tsx:48]: ah was about to comment about this but makes sense. thanks for clarifying!

File: src/screens/sync/BubbleSync.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { useContext } from "react";
+import { Image, Dimensions } from "react-native";
+import { SyncContext } from "../../contexts/SyncContext";
+
+export const BubbleSync = ({ show }: { show: boolean }) => {
+  const [allSyncs] = useContext(SyncContext);
+
+  const width = Dimensions.get("window").width;
+  const height = Math.round((width * 620) / 371);
+
```
Comment (achou11) [inline src/screens/sync/BubbleSync.tsx]: both `width` and `height` can be defined outside the component

**Review by achou11 — APPROVED** [review]

**Pull request merged**